packages/strapi-design-system/src/SimpleNav/SimpleNavItem.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { BaseLink } from '../BaseLink';
+import { Box } from '../Box';
+import { Flex } from '../Flex';
+import { Typography } from '../Typography';
+import { buttonFocusStyle } from '../themes/utils';
+
+const SimpleNavItemWrapper = styled(Box)`
+  text-decoration: none;
+  background: ${({ background, theme }) => theme.colors[background] || background};
+  color: ${({ textColor, theme, withError }) => theme.colors[withError ? 'danger600' : textColor]};
+
+  &.active,
+  &:active,
+  &[aria-current='page'] {
+    background: ${({ theme, withError }) => theme.colors[withError ? 'danger100' : 'primary100']};
+    color: ${({ theme, withError }) => theme.colors[withError ? 'danger600' : 'primary700']};
+  }
+
+  &:not(.active):not(:active):not([aria-current='page']) {
+    &:hover {
+      background: ${({ theme, withError }) => theme.colors[withError ? 'danger100' : 'neutral150']};
+      color: ${({ theme, withError }) => theme.colors[withError ? 'danger600' : 'neutral800']};
+    }
+  }
+
+  svg,
+  svg * {
+    fill: currentColor;
+  }
+
+  ${buttonFocusStyle}
+`;
+
+const SimpleNavItemIconWrapper = (props) => <Flex as="span" alignItems="center" aria-hidden="true" {...props} />;
+
+export const SimpleNavItem = ({
+  children,
+  className,
+  endIcon,
+  gap,
+  isActive,
+  startIcon,
+  textVariant,
+  withError,
+  ...rest
+}) => {
+  return (
+    <SimpleNavItemWrapper className={`${className}${!isActive ? '' : ' active'}`} withError={withError} {...rest}>
+      <Typography variant={textVariant} textColor="currentColor">
+        <Flex as="span" display="flex" alignItems="center" justifyContent="space-around" gap={gap} width="100%">
+          {startIcon && <SimpleNavItemIconWrapper>{startIcon}</SimpleNavItemIconWrapper>}
```

Comment:
IMHO I really don't like the pattern of `startIcon` and `endIcon` and whilst I do appreciate we use it elsewhere in the design system I'm not sure it makes that much sense, you're then assuming that all Icons need to behave the same which we've found that they don't i.e. the sizing / spacing is different per use case according to the design.

I would just leave it as `children` and let the user do it – Composability.